src/components/pokemonSelectorModal.html

```diff
@@ -87,21 +87,25 @@ <h5 class="modal-title text-light">Select a Pokémon</h5>&nbsp;
                   <thead>
                     <tr class="text-light">
                       <th class="text-left bg-dark align-middle">Pokémon</th>
-                      <th class="text-center bg-dark align-middle">Proteins Used</th>
+                      <th class="text-center bg-dark align-middle">Vitamins Used</th>
+                      <th class="text-center bg-dark align-middle" data-bind="text: `${GameConstants.VitaminType[VitaminController.currentlySelected()]} Used`">X Used</th>
                       <th class="text-center bg-dark align-middle">Attack Bonus</th>
+                      <th class="text-center bg-dark align-middle">Egg Steps</th>
                     </tr>
                   </thead>
-                  <tbody data-bind="foreach: PartyController.getProteinSortedList()">
-                    <tr class="clickable" data-bind="click: () => $data.useProtein(VitaminController.getMultiplier()), hidden: $data.hideFromProteinList()">
+                  <tbody data-bind="foreach: PartyController.getvitaminSortedList()">
+                    <tr class="clickable" data-bind="click: () => $data.useVitamin(VitaminController.currentlySelected(), VitaminController.getMultiplier()), attr: { onContextMenu: `App.game.party.getPokemonByName('${$data.name}')?.removeVitamin(VitaminController.currentlySelected(), VitaminController.getMultiplier())` }, hidden: $data.hideFromProteinList()">
```

Comment:
Can we get the leftclick to stop the input after it's done? For us that have disabled the "block leftclick" setting? I think it's usually just `return false;` at the end

Reply:
I'm not quite sure what you mean, we only have a disable right click option AFAIK.

Left click adds, right click removes _(even with disable right click enabled)_

Reply:
I switched left and right in my explanation...

Reply:
I'm still not entirely sure what you mean by "Can we get the right click to stop the input after it's done?"

Reply:
Jaaslet wants to prevent the context menu from opening even when the related setting is disabled.

Reply:
Ah I see, thanks, my brain just wasn't clicking 😅

Reply:
Resolved, also added text stating `Left click to add, right click to remove`